posts/2021-10-12-open-beta-changes.md

```diff
@@ -9,7 +9,12 @@ authors:
   - Brian McNamara
 ---
 
-# Overview
+## Updates
```

Comment:
Why the change in header value?

Reply:
The first H1 is stripped, the others remain.  I demoted all so they all show up :).